### PR TITLE
Amend salaried teacher training non-UK content

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/ffcmncjF/6458

### CODE REVIEW

**Comment by emmaachesongray** [comment]
@Sarah-DfE please could you 2i this?

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md [hunk 1]
```diff
@@ -48,6 +48,15 @@ expander:
       <p>
       Learn more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.
       </p>
+  salaried-teacher-training:
+    title: salaried teacher training
+    text: |-
+      <p>
+      You can apply for a salaried teacher training course. However, these courses are limited in number and very competitive.
```
Comment (Sarah-DfE) [inline app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md:55]: Just seeing if we need 'in number', we don't have it in the sentence below, just thinking we don't need it here either?

**Review by Sarah-DfE — DISMISSED** [review]
Just one comment, but other than that this looks good to go!

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md (hunk 1 above)
Comment (emmaachesongray) [reply app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md:55]:
```suggestion
      You can apply for a salaried teacher training course. However, places on these courses are limited and very competitive.
```

**Pull request merged**